ticdc/ticdc-overview.md

```diff
@@ -87,6 +87,61 @@ As shown in the architecture diagram, TiCDC supports replicating data to TiDB, M
 
+## Implementation of data change processing
+
+This section mainly describes how TiCDC processes data changes generated by upstream DML operations.
+
+For data changes generated by upstream DDL operations, TiCDC obtains the complete DDL SQL statement, converts it into the corresponding format based on the sink type of the downstream, and sends it to the downstream. This section does not elaborate on this.
+
+> **Note:**
+>
+> The logic of how TiCDC processes data changes might be adjusted in subsequent versions.
+
+MySQL binlog directly records all DML SQL statements executed in the upstream. Unlike MySQL, TiCDC listens to the real-time information of each Region Raft Log in the upstream TiKV, and generates data change information based on the difference between the data before and after each transaction, which corresponds to multiple SQL statements. TiCDC only guarantees that the output change events are equivalent to the changes in the upstream TiDB, and does not guarantee that it can accurately restore the SQL statements that caused the data changes in the upstream TiDB.
+
+Data change information includes the data change types and the data values before and after the change. The difference between the data before and after the transaction can result in three types of events:
+
+1. The `DELETE` event: corresponds to a `DELETE` type data change message, which contains the data before the change.
+
+2. The `INSERT` event: corresponds to a `PUT` type data change message, which contains the data after the change.
+
+3. The `UPDATE` event: corresponds to a `PUT` type data change message, which contains the data both before and after the change.
+
+Based on the data change information, TiCDC generates data in the appropriate formats for different types of downstream, and sends the data to the downstream. For example, it generates data in formats such as Canal-JSON and Avro and writes the data to Kafka, or converts the data back into SQL statements and sends them to the downstream MySQL or TiDB.
+
+Currently, when TiCDC adapts data change information for the corresponding protocol, for specific `UPDATE` events, it might split them into one `DELETE` event and one `INSERT` event. For more information, see [Split update events into delete and insert events](/ticdc/ticdc-behavior-change.md#split-update-events-into-delete-and-insert-events).
+
+When the downstream is MySQL or TiDB, TiCDC cannot guarantee that the SQL statements written to the downstream are exactly the same as the SQL statements executed in the upstream. This is because TiCDC does not directly obtain the original DML statements executed in the upstream, but generates SQL statements based on the data change information. However, it ensures the consistency of the final results.
```

Comment:
```suggestion
When the downstream is MySQL or TiDB, TiCDC cannot guarantee that the SQL statements written to the downstream are exactly the same as the SQL statements executed in the upstream. This is because TiCDC does not directly obtain the original DML statements executed in the upstream, but generates SQL statements based on the data change information. However, TiCDC ensures the consistency of the final results.
```